src/spikeinterface/core/sortinganalyzer.py

```diff
@@ -219,6 +219,9 @@ def __init__(
         # this is used to store temporary recording
         self._temporary_recording = None
 
+        # for zarr format, we store the kwargs to create zarr datasets (e.g., compression)
+        self._zarr_kwargs = {}
```

Comment:
why not using the storage option ?

Reply:
what about `_backend_kwargs`? Storage option could be one of those, but it's not the same. It's a keyword argument for the `zarr.open`